### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by Polfo** [Laravel-Backpack/devtools-issues#23]
# Feature Request

### What's the feature you think DevTools should have?

In http://127.0.0.1:8000/admin/devtools/model
When I use the Generate button, aparantly nothing happens. It can take a minute before the operation is done. 
Could there be immediate 'processing request' feedback?

### How often do you see yourself using this feature, if implemented?

Every time I click the button

**Comment by tabacitu** [comment]
Hmm.... normally it should work very fast. You said you're running this on `php artisan server` right, on a Windows 10 machine? Can you give us the specs of the machine please (CPU, RAM)?

@promatik can you please test performance under `php artisan serve` on Windows 10? And compare with your normal webserver setup?

**Comment by Polfo** [comment]
> Hmm.... normally it should work very fast. You said you're running this on `php artisan server` right, on a Windows 10 machine? Can you give us the specs of the machine please (CPU, RAM)?

@tabacitu 
AMD A8-5600K APU with Radeon(tm) HD Graphics 3.60 GHz
RAM 8,00 GB

**Comment by Polfo** [comment]
`php artisan serve` has a similar performance to the `laragon` setup with `apache` on my machine

**Comment by promatik** [comment]
@tabacitu as @Polfo said, yes, laragon as similar performance to artisan.
In fact, the generate button takes a little, even in my machine, it can take up to 10 seconds. For me this was never a problem, I always noticed the google chrome loading bar.

![image](https://user-images.githubusercontent.com/1838187/145917762-0ba0729e-6236-459c-84ee-fcb3e30e9fec.png)

But yes, we may add a notification when the process starts. @tabacitu does this makes sense?

**Comment by tabacitu** [comment]
Agreed. Let's add a full-page loading sign when the form is being submitted. Nothing clearer than that 😀 Any way to do this in 4.1 without being too intrusive @promatik ? The best way I see would be to do `$this->crud->setCreateView('your-view')` to point to a custom view, that includes the initial view and pushes a script.

I'm wondering, is this worth doing in 4.1 or can we just postpone until we launch 4.2 later this month? Because after Backpack 4.2 is launched, we can easily use the `script` widget to create that loading sign, without overriding any views.

I say let's postpone until we launch 4.2.

**Comment by tabacitu** [comment]
Note to self - in v5 we now have the `script` widget. We can use that to push a bit of JS on the two forms, so that when the form is submitted, we show a nice overlay with a loading sign or "_please wait..._" message.
